src/components/ControllerTop.vue

```diff
@@ -387,16 +418,27 @@ export default {
     };
   },
   mounted() {
-    this.fetchKeyboards().then(() => {
+    console.info('mounted start');
+    this.initializeKeyboards().then(() => {
```

Comment:
hmm, I wonder if we can't set this from props into the component from App.vue

Reply:
MMM both way works in fact. Props may be more "component" way yes.